bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.0.4
+version: 11.0.5
```

Comment:
Sorry for not pointing this out before, you need to bump the version to a new minor `11.1.0` as you are including new parameters to the chart.

Reply:
Ok.I will update it later.

Reply:
Ok.I will update it later.